src/test.js

```diff
@@ -36,6 +36,7 @@ export default function Test( settings ) {
 	this.module = config.currentModule;
 	this.stack = sourceFromStacktrace( 3 );
 	this.steps = [];
+	this.timeout = 0;
```

Comment:
I didn't put much thought into this when I was implementing it originally. `undefined` seems reasonable and is consistent with the current `testTimeout`, so I will go that route.